shelley/chain-and-ledger/executable-spec/src/Value.hs

```diff
@@ -0,0 +1,67 @@
+{-# LANGUAGE DerivingVia #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE DeriveGeneric #-}
+--{-# LANGUAGE DeriveAnyClass #-}
+{-# LANGUAGE StandaloneDeriving #-}
+{-# LANGUAGE PatternSynonyms #-}
+{-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE OverloadedStrings #-}
+
+module Value
```

Comment:
please add some comment of what this represents / how it works

Reply:
I forgot to remove the Value module - all the code in it had to go into Scripts (because of cycle dependecies). I will remove this module and put all the relevant comments in Scripts (where this code lives now)